phpBB/includes/functions.php

```diff
@@ -2130,7 +2130,7 @@ function check_form_key($form_name, $timespan = false)
 /**
 * Build Confirm box
 * @param boolean $check True for checking if confirmed (without any additional parameters) and false for displaying the confirm box
-* @param string $title Title/Message used for confirm box.
+* @param string|array $title Title/Message used for confirm box.
```

Comment:
Please extend this to also state how the array version works, i..e how the array should be structured to work.